### PR TITLE
Update all usage of Tumblr's CSS variables

### PR DESCRIPTION
**Pull request by AprilSylph**
most notably fixes AlreadyReblogged, Tag Tracking+ being readable on dark mode, and Reblog Timestamps being the right colour

let me know if it would be better to split this up per extension

### CODE REVIEW

**Review by hobinjk — APPROVED** [review]
Don't have the time to test locally but code LGTM

File: Extensions/blacklist.js
Comment (AprilSylph) [reply Extensions/blacklist.js:182]: I was worried that it wouldn't work, but 1. `var()` only allows one fallback value, and 2. MDN specifies that the only illegal characters in the fallback value are "newlines, unmatched closing brackets, i.e. `)`, `]`, or `}`, top-level semicolons, or exclamation marks". I'll admit that Blacklist is the only thing here that I didn't test though, so one of us probably should do that before merging

File: Extensions/tweaks.js
Comment (AprilSylph) [reply Extensions/tweaks.js:810]: genuinely didn't even notice this. yay for atom+editorconfig

**Comment by AprilSylph** [comment]
i'm gonna hold off on merging this until monday morning because ~~i don't want to potentially mess up my neat contribution graph~~ if any issues arise from this i'd prefer to be able to jump right on them

**Pull request merged**
